Tensile/Source/TensileCreateLibrary.cmake

```diff
@@ -30,6 +30,7 @@ function(TensileCreateLibraryCmake
     Tensile_COMPILER
     Tensile_CODE_OBJECT_VERSION
     Tensile_ARCHITECTURE
+    Tensile_LIBRARY_FORMAT
     Tensile_MERGE_FILES
     Tensile_SHORT_FILE_NAMES
     Tensile_LIBRARY_PRINT_DEBUG )
```

Comment:
I'll also point out that you are changing the order of arguments to the function CreateLibraryCmake so all callers must modify their calls.   Note the comment in ARGN handling below, about breaking the old API, it is easy enough to update rocBLAS but not sure about other callers outside tensile.